src/panels/config/application_credentials/dialog-add-application-credential.ts

```diff
@@ -165,20 +166,19 @@ export class DialogAddApplicationCredential extends LitElement {
                   : nothing}
                 ${this._params.selectedDomain
                   ? nothing
-                  : html`<ha-combo-box
+                  : html`<ha-generic-picker
```

Comment:
When adding from scratch, or there are no extra instructions, this shows up like this:

<img width="830" height="1126" alt="Image" src="https://github.com/user-attachments/assets/bf8bc59b-6a4f-4fcd-ae67-6510e5683744" />
<img width="886" height="538" alt="image" src="https://github.com/user-attachments/assets/aedb0093-cbec-4ce0-8430-ec2953b36350" />